docs/topics/api/overview.rst

```diff
@@ -27,6 +27,12 @@ There are some common API responses that you can expect to receive at times.
 
 .. http:get:: /api/v3/...
 
+    :statuscode 200: Success.
+    :statuscode 201: Creation successful.
+    :statuscode 202: The request has been accepted for processing.
```

Comment:
Although that's specified somewhere else I believe we should add something like "This usually executes one or more asyncronous tasks so that results aren't visible immediately." - or something similar. We only use it once in the signing API for now but especially there the results aren't visisble imediately.